src/shared/utilities/childProcess.ts

```diff
@@ -152,7 +153,11 @@ export class ChildProcess {
 
         // Emitted when streams are closed.
         this.childProcess.once('close', (code, signal) => {
-            const result = this.makeResult(code)
+            if (typeof code !== 'number') {
+                getLogger().debug(`child_process: terminated by signal '${signal}'`)
```

Comment:
use `this.log` instead of getLogger(). Should this be at least a warn() or even error()? I would not expect it to be common.

Reply:
Looks like it only happens right now when the lambda invoker times out. Will change to a warn.